contents/docs/feature-flags/cutting-costs.mdx

```diff
@@ -3,23 +3,47 @@ title: Cutting feature flag costs
 sidebar: Docs
 showTitle: true
 ---
-import PostHogBillableUsageTemplate from "../_snippets/posthog-billable-usage-template.mdx"
+import { CalloutBox } from 'components/Docs/CalloutBox'
+
+We aim to be significantly cheaper than our competitors. In addition to our [pay-as-you-go pricing](/pricing), below are tips to reduce your feature flag costs.
+
+## Understanding your bill
+
+Feature flags are charged based requests made to our `/flags` endpoint, which is used to evaluate a feature flag for a given user. They are not billed based on `$feature_flag_called` events, which are optional events sent for metrics tracking, not for actual feature flag evaluation. 
+
+The number of `/flags` requests and the number of `$feature_flag_called` events are **not** directly related.
 
-We aim to be significantly cheaper than our competitors. In addition to our [pay-as-you-go pricing](/pricing), below are tips to reduce your feature flag costs:
+### Creating a billable usage dashboard
+
+import PostHogBillableUsageTemplate from "../_snippets/posthog-billable-usage-template.mdx"
 
 <PostHogBillableUsageTemplate />
 
-## Reduce client-side feature flag requests
+import FlagChargeEstimate from "./snippets/flag-charge-estimate.mdx"
+
+<details>
+  <summary>Estimating your bill manually (with math!)</summary>
+
+  <FlagChargeEstimate />
+
+</details>
+
+
+## Reducing client-side feature flag requests
 
 In our [JavaScript Web](/docs/libraries/js) and [React Native](/docs/libraries/react-native) SDKs, you can reduce the cost of feature flags by reducing the number of requests your client makes to fetch feature flag values for your users.
 
 1. Using the the [advanced configuration](/docs/libraries/js/config#advanced-configuration), set `advanced_disable_feature_flags_on_first_load` to `true`. This stops automatically requesting feature flags whenever a page loads. This is useful when you're immediately calling `posthog.identify()` anyway, which requests flags again.
 
 2. Using the the [advanced configuration](/docs/libraries/js/config#advanced-configuration), set `advanced_disable_feature_flags` to `true`. This stops PostHog automatically requesting feature flags. Instead, use [bootstrapping](/docs/feature-flags/bootstrapping) to load flags exactly once.
 
-> **Note:** Use of `advanced_disable_feature_flags` will disable surveys for all users, as surveys rely on feature flags internally.
+<CalloutBox icon="IconWarning" title="Note" type="caution">
+
+  Use of `advanced_disable_feature_flags` will disable surveys for all users, as surveys rely on feature flags internally.
+
+</CalloutBox>
```

Comment:
this is a good callout. 

Something else I would add is that there's an additional configuration called `advanced_only_evaluate_survey_feature_flags`, which, when true, _only_ evaluates survey feature flags.  Useful if someone has feature flags that they're using in the backend but they don't want to get charged for using them on the frontend.